tests/components/edl21/test_config_flow.py

```diff
@@ -0,0 +1,81 @@
+"""Test EDL21 config flow."""
+
+import pytest
+
+from homeassistant import data_entry_flow
```

Comment:
It's a small style thing, but I think it's better to only import `FlowResultType`:
```python
from homeassistant.data_entry_flow import FlowResultType
```